### ISSUE TITLE
Add societal benefit area object information

### ISSUE THREAD

**Issue by rmarow** [nsidc/usaon-vta-docs#19]
There isn't currently a section in the glossary describing societal benefit area objects

**Comment by MattF-NSIDC** [comment]
@rmarow can you provide more detail about what questions need to be answered here?

**Comment by hazelshapiro** [comment]
There isn't a section in the glossary describing `societal benefit area objects` - I need to add that information.

**Comment by rmarow** [comment]
https://docs.google.com/spreadsheets/d/1pTvFVg-7YsJLqJ81YsMXwustwN28x-Jfkce9Znx0bHM/edit#gid=0

**Comment by rmarow** [comment]
@hazelshapiro is there any ability to edit that spreadsheet? I noticed that the way this is done is merging the cells on the left and that prevents the ability to use filtering for easier viewing.

**Comment by hazelshapiro** [comment]
Definitely. I've added you as an editor. Could you copy this info to a second tab and edit there, though, just so we also preserve this formatting? How are you planning to use this sheet?
